packages/react-pdf-toolkit/.storybook/main.ts

```diff
@@ -0,0 +1,20 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+const config: StorybookConfig = {
+  core: {
+    builder: '@storybook/builder-vite', // 👈 The builder enabled here.
+  },
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+  ],
+  framework: {
+    name: '@storybook/react-vite',
+    options: {},
+  },
+  docs: {
+    autodocs: 'tag',
+  },
+};
+export default config;
```

Comment:
Please add spacing and try to find an opportunity to add the monorepo's Prettier and ESLint to this package. 🙏🏼